packages/lib-content/src/components/AboutHeader/AboutHeader.js

```diff
@@ -51,11 +67,11 @@ function AboutHeader({ pathname = '' }) {
         >
           {links.map(link => (
             <Box as='li' key={link.label} pad={{ right: 'small' }}>
-              <NavLink pathname={pathname} color='white' label={link.label} href={link.href} />
+              <NavLink color='white' label={link.label} href={link.href} />
             </Box>
           ))}
-        </Box>
-        <ZooniverseLogotype id="About Zooniverse Header component" color='white' />
+        </StyledBox>
+        <StyledLogo id='About Zooniverse Header component' color='white' />
```

Comment:
This `id` gets passed to an individual SVG title id attribute. When I checked the StyledLogo tests it looks like "zooniverse-logo" would be more appropriate.

Reply:
Good point! I'll fix the ids